x-pack/test/apm_api_integration/tests/mobile/mobile_detailed_statistics_by_field.spec.ts

```diff
@@ -110,8 +111,16 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           });
         });
         it('returns some data for both periods', async () => {
-          expect(isEmpty(mobiledetailedStatisticResponse.currentPeriod)).to.be.equal(false);
-          expect(isEmpty(mobiledetailedStatisticResponse.previousPeriod)).to.be.equal(false);
+          expect(Object.keys(mobiledetailedStatisticResponse.currentPeriod)).to.be.eql([
+            '2.3',
+            '1.1',
+            '1.2',
```

Comment:
Why are these strings and not numbers?

Reply:
These are the service.version, it is a string